frontend/src/views/SecretOverviewPage/components/CreateSecretForm/CreateSecretForm.tsx

```diff
@@ -59,6 +62,13 @@ export const CreateSecretForm = ({
     formState: { isSubmitting, errors }
   } = useForm<TFormSchema>({ resolver: zodResolver(typeSchema) });
   const newSecretKey = watch("key");
+  const newValue = watch("value");
```

Comment:
nit: let's be more specific with the name? maybe "newSecretValue"

Reply:
thanks, made the change